### PR TITLE
[KBFS-3235] Widget: Data models and connectors for recent files rows

### PR DESCRIPTION
**Pull request by jzila**
Still not ready

### CODE REVIEW

**Comment by jzila** [comment]
ping--need a thumb here

File: shared/actions/fs/index.js [hunk 1]
```diff
@@ -75,6 +75,19 @@ const filePreview = (state: TypedState, action) =>
     )
     .catch(makeRetriableErrorHandler(action))
 
+const loadUserFileEdits = (state: TypedState, action) =>
+  RPCTypes.SimpleFSSimpleFSUserEditHistoryRpcPromise()
+    .then(writerEdits => {
+      const tlfUpdates = Constants.userTlfHistoryRPCToState(writerEdits || [])
+      Saga.all(tlfUpdates.map(u =>
```
Comment (chrisnojima) [inline shared/actions/fs/index.js]: i think this line does nothing

Comment (jzila) [reply shared/actions/fs/index.js]: Yeah it's fixed in the next PR.

Comment (jzila) [reply shared/actions/fs/index.js]: Oh I see what you mean. I guess I need to chain it.

**Pull request merged**